app/views/support_interface/change_course/options.html.erb

```diff
@@ -1,24 +1,28 @@
 <% content_for :browser_title, t('support_interface.change_course.page_title') %>
```

Comment:
Title on this page should contain `Error:` if the form below fails validation; there should be a helper we can use for this.